Sources/Plasma/PubUtilLib/inc/CMakeLists.txt

```diff
@@ -33,7 +34,4 @@ set_property(
         plSurface
         plVault
 )
-set_property(
-    TARGET plPubUtilInc
-    PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}"
-)
+target_include_directories(plPubUtilInc INTERFACE "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
Do we want `source_group("Header Files" FILES ${plPubUtilInt_HEADERS})` for folders in VS & Xcode?

Reply:
CMake automatically puts headers into that group if not groups are specified.

![image](https://user-images.githubusercontent.com/714455/181935000-9c1082f1-70cb-4370-a43c-86689164accf.png)

Reply:
Well that's not obvious behaviour, but okay...